easybuild/easyconfigs/a/augur/augur-7.0.2-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,130 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonBundle'
+
+name = 'augur'
+version = '7.0.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/nextstrain/augur'
+description = "Pipeline components for real-time phylodynamic analysis"
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('Biopython', '1.75', versionsuffix),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('MAFFT', '7.453', '-with-extensions'),
+    ('IQ-TREE', '1.6.12'),
+    ('VCFtools', '0.1.16'),
+    ('RAxML', '8.2.12', '-hybrid-avx2'),
+    ('FastTree', '2.1.11'),
+    ('CVXOPT', '1.1.9', versionsuffix),
+    ('matplotlib', '2.2.4', versionsuffix),
```

Comment:
I suspect that `matplotlib` 3.x would work just fine as well, despite the script version requirements...

Polled for info via https://github.com/nextstrain/augur/issues/538